docs/usage.rst

```diff
@@ -267,17 +267,33 @@ By default, Raven does not capture unhandled promise rejections. You can have it
     captureUnhandledRejections: true
   }).install();
 
-Global Error Handler
---------------------
+Global Fatal Error Handler
+--------------------------
 
-The ``install`` method sets up a global listener for uncaught exceptions, and possibly
-also for unhandled rejections. You generally shouldn't carry on after receiving an `uncaughtException`
-(see `Node docs <http://nodejs.org/api/process.html#process_event_uncaughtexception>`_),
-so you can provide a callback which will be invoked **after** Raven has sent the event to Sentry:
+The ``install`` method sets up a global listener for uncaught exceptions, and ``context`` and ``wrap`` can catch exceptions as well.
+These are situations where Raven catches what would otherwise be a fatal process-ending exception, and we generally shouldn't keep our
+program running after such events (see `Node docs <http://nodejs.org/api/process.html#process_event_uncaughtexception>`_),
+so Raven has a concept of a "fatal error handler". When Raven catches what would otherwise be a fatal process-ending exception, we
+capture it and then call the fatal error handler.
```

Comment:
this we is fine IMO because it's referring to sentry authors. but consider changing to "raven will capture it"